### ISSUE TITLE
should report error for identical method names

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2089]
```abap
INTERFACE if_ixml_node.
  METHODS:
    append_child IMPORTING new_child TYPE REF TO if_ixml_element,
    append_child IMPORTING child TYPE REF TO if_ixml_node.
ENDINTERFACE.
```

**Comment by larshp** [comment]
guess it only works for classes ?

![image](https://user-images.githubusercontent.com/5888506/129908253-3fcc446d-e538-4185-bfcd-19a34cb12883.png)
